### DISCUSSION TITLE
Governor: Sei parameters

### DISCUSSION THREAD

**Discussion by tbjump** [wormhole-foundation/wormhole#2793] [Governance] [answered]
I want to open up a discussion for setting Governor parameters for Sei (not to be confused with Sui). 

Guardians should agree on a default value for the
* maximum accumulative instant transaction notional value in 24h rolling window (dailyLimit)
* maximum instant transaction notional value (bigTransactionSize)

The objective is to
* set these parameters high enough to support legitimate demand
* set these parameters low enough to make L1 risk and smart contract risk reasonable

In the past, most new chains started with a configuration of $200k/$20k and were adjusted up depending on actual demand.

I am proposing the standard $200k/$20k parameters for Sei.

**Comment by brianlong** [comment, marked as answer]
I see no reason to deviate from our standard params in the beginning.
